docs/_data/prague-2020-schedule.yaml

```diff
@@ -0,0 +1,77 @@
+writing_day:
+  - time: '9:00'
+    title: 'Platform opens'
+  - time: '10:00'
+    title: Morning introduction - Introduction to Sprints & Project Introductions
+  - time: '10:15'
+    title: Team up and start working
+  - time: '13:30'
+    title: Lunch Break
+  - time: '18:00'
+    title: End of Writing Day
+
+talks_day1:
+  - time: '9:00'
+    title: 'Platform opens'
+  - time: '10:00'
+    title: Write the Docs Team - Introduction to Write the Docs
+  - time: '10:45'
+    title: "<b>Unconference Starts</b>"
+  - time: '10:45'
+    slug: from-graffiti-writer-to-technical-writer-kruno-golubi
+  - time: '11:30'
+    slug: organizing-a-confluence-hoard-or-does-this-page-spark-joy-abigail-sutherland
+  - time: '12:15'
+    slug: documenting-the-ancient-history-of-your-project-natali-vlatko
+  - time: '13:00'
+    slug: the-baseline-or-technical-writing-for-non-technical-readers-paul-brown
+  - time: '13:45'
+    title: Lunch (60 mins)
```

Comment:
This feels a little late to me -- should we swap one talk after lunch, and do lunch at 13:00?

Reply:
Yeah, makes sense. :/

Reply:
What about the return of the snack break? Maybe we can add 15min after the first 2 talks (well, 3 talks including the conference intro) and move lunch 15min later?

Reply:
That seems to move lunch later, no earlier?

Reply:
Yeah, I think it's best to keep 45min slots for now coz it might mess up the schedule too much. Maybe bump one talk to after lunch and add the 15min snack break in the afternoon instead of the morning?

Reply:
Alright folks, lunch switched.
Snack break seems less important cos folks can just grab stuff from their own kitchen.
I'm calling this done.